src/sardana/taurus/qt/qtgui/extra_sardana/qtspock.py

```diff
@@ -146,8 +146,9 @@ class QtSpockWidget(RichJupyterWidget, TaurusBaseWidget):
         from taurus.external.qt import Qt
         from sardana.taurus.qt.qtgui.extra_sardana.qtspock import QtSpockWidget
         app = Qt.QApplication([])
```

Comment:
```suggestion
        app = Qt.QApplication(["qtspock"])
```

I just tried it on Debian Buster with PyQt v5.11.3 and with Qt 5.11.3 and the Qt.QApplication requires a list with a string in order to not crash. We hit the same thing in the past already with spock https://github.com/sardana-org/sardana/issues/1209. I imagine newer versions do not have this problem but maybe it is better to pass a string to be on the safe side.